ParticleView/src/main/java/me/wangyuwei/particleview/ParticleView.java

```diff
@@ -179,7 +192,7 @@ private void startParticleAnim() {
         Collection<Animator> animList = new ArrayList<>();
 
         ValueAnimator textAnim = ValueAnimator.ofInt(DEFAULT_MAX_TEXT_SIZE, mParticleTextSize);
-        textAnim.setDuration(800);
+        textAnim.setDuration((int)(mTextAnimTime * 0.8));
```

Comment:
@drivfe @JeasonWong Since we are using set.playTogether(animList); later, the fix will keep proportional to the original example. I've noticed one more thing maybe is better. include too later

`animator.setDuration(mTextAnimTime + (mTextAnimTime * 0.02) * i + (mTextAnimTime * 0.03) * j);`

Let me know if it makes sense.